### ISSUE TITLE
Redisearch aliasing of results is returning undefined node error

### ISSUE THREAD

**Issue by ChrisSandison** [RedisGraph/RedisGraph#1533]
Using docker-image version `v2.0.21` and use case fulltext example given here https://oss.redislabs.com/redisgraph/commands/#full-text-indexes

See example below

```
127.0.0.1:6379> graph.query demo "CALL db.idx.fulltext.createNodeIndex('Movie', 'title')"
1) (empty array)
2) (empty array)
3) 1) "Indices created: 1"
   2) "Query internal execution time: 0.514054 milliseconds"
127.0.0.1:6379> graph.query demo "CREATE (:Movie {title: 'Jungle Book'})"
1) 1) "Nodes created: 1"
   2) "Properties set: 1"
   3) "Query internal execution time: 0.463152 milliseconds"
127.0.0.1:6379> graph.query demo "CALL db.idx.fulltext.queryNodes('Movie', 'Book') YIELD node RETURN node.title"
1) 1) "node.title"
2) 1) 1) "Jungle Book"
3) 1) "Query internal execution time: 0.308680 milliseconds"
127.0.0.1:6379> graph.query demo "CALL db.idx.fulltext.queryNodes('Movie', 'Book') YIELD node AS m RETURN m.title"
(error) node not defined
```

**Comment by jeffreylovitz** [comment]
Hi @ChrisSandison,

This issue has been fixed on more recent versions of RedisGraph. Can you switch to the Docker tag `2.2.12` or similar?

**Comment by ChrisSandison** [comment]
Hey @jeffreylovitz 

Is there plans on backporting this fix? We are running v2.0.10 and this is functional. Additionally, this is an example taken from your documentation.
